recipes/slow-buffer-to-buffer-alloc-unsafe-slow/src/workflow.ts

```diff
@@ -0,0 +1,291 @@
+import type { Edit, SgRoot, SgNode, Kinds } from "@codemod.com/jssg-types/main";
+import type Js from "@codemod.com/jssg-types/langs/javascript";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+
+export default function transform(root: SgRoot): string | null {
+	const rootNode = root.root();
+	const edits: Edit[] = [];
+	let hasChanges = false;
```

Comment:
I think it's still on our contribution.md, but you can remove `hasChanges` logic to use `edits.lenght`